src/renderer/components/side-nav/side-nav.vue

```diff
@@ -104,11 +104,16 @@
         @click="goToChannel(channel.id)"
       >
         <div
-          class="thumbnailContainer"
+          v-if="!hideActiveSubscriptions"
```

Comment:
This new `<div>` container here doesn't seem to be needed. With this current logic, Vue will still do a loop on all of the channels in the subscription list, but will display nothing. Instead, we should make it so that the loop doesn't run at all if the new setting is enabled.

Simply adding the `if` logic to the existing `<div>` will accomplish this. This is the only line that needs to be added to this file.